Search.jsp

```diff
@@ -121,9 +124,9 @@ footer
 <header>
 <nav>
 <a href="Controller?page=home">Home</a>
-<a href="Controller?page=aboutus">About us</a>
-<a href="Controller?page=contact">Contact</a>
-<a href="Controller?page=admin">Admin</a>
+<!--  <a href="Controller?page=aboutus">About us</a> -->
+<a href="Controller?page=contact">Help & Contact</a>

```

Comment:
Use "**Contact us for help**".

**Hui**

Reply:
OK